graph/typeDefs.graphql

```diff
@@ -730,8 +751,8 @@ type RootQuery {
   # role.
   me: User
 
-  # Users returned based on a query.
-  users(query: UsersQuery): [User]
+  # Users returned based on a query. Requires the `ADMIN` role.
```

Comment:
Shouldn't need to indicate role here.

Reply:
Well yeah but you need special privilege to access it, not sure how to describe that. (Just copied what we used before)

Reply:
Requires the `SEARCH_OTHER_USERS` permission?

Reply:
I would suggest we remove all the role requirements from the comments to be honest until we can get custom annotations.

Reply:
Custom annotations, is that a thing?